src/ucp/core/ucp_mm.c

```diff
@@ -437,6 +437,32 @@ static void ucp_memh_cleanup(ucp_context_h context, ucp_mem_h memh)
     ucs_free(memh);
 }
 
+ucs_status_t ucp_memh_register_one(ucp_context_h context, ucp_mem_h memh,
+                                   void *address, size_t length,
+                                   ucp_md_index_t md_index,
+                                   uct_md_mem_reg_params_t *reg_params)
+{
+    size_t reg_length, reg_align;
+    void *reg_address;
+
+    if (context->rcache == NULL) {
+        reg_align   = ucs_max(context->tl_mds[md_index].attr.reg_alignment,
+                              UCS_RCACHE_MIN_ALIGNMENT);
```

Comment:
why is this value used as a minimum?

Reply:
It's minimum supported by pgtable implementation, same logic in `ucp_memh_reg_align()`

Reply:
isn't it only relevant for rcache use-case?

Reply:
Regarding `ucp_memh_reg_align()`, can you pls also check why [this assert](https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_mm.c#L768) does not fail when rcache is disabled?